.env.test

```diff
@@ -1,3 +1,4 @@
 NODE_ENV=test
 UPLOAD_SERVICE_URL=http://localhost:3000
 PAYMENT_SERVICE_URL=http://localhost:4000
+ARWEAVE_GATEWAY=http://arlocal:1984
```

Comment:
this one should be `localhost` - only those in the docker-file should use the aliases